pkg/vm/json_test.go

```diff
@@ -150,8 +149,17 @@ func testFile(t *testing.T, filename string) {
 	data, err := ioutil.ReadFile(filename)
 	require.NoError(t, err)
 
+	// get rid of possible BOM
+	if len(data) > 2 && data[0] == 0xef && data[1] == 0xbb && data[2] == 0xbf {
+		data = data[3:]
+	}
+	if strings.HasSuffix(filename, "MEMCPY.json") {
+		return // FIXME this file is not a valid JSON
```

Comment:
There should be a neo-vm bug for it.

Reply:
https://github.com/neo-project/neo-vm/issues/322